ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -358,11 +359,13 @@ def fetch_views(
                 continue
 
     def _parse_data_model(self):
-        logger.info("Parsing Data Models")
+
         if (
             self.config.dbt_manifest_file is not None
             and self.config.dbt_catalog_file is not None
```

Comment:
We could change this for
```python
if (self.config.dbt_manifest_file and self.config.dbt_catalog_file)
```
To use the full `str` truth value and discard empty strings